Documentation/GeneralConventions/ReviewPolicy.rst

```diff
@@ -0,0 +1,201 @@
+    nicely and give feedback on problematic areas of a Pull Request and ask
+    the contributor to rephrase or fix errors. Other maintainers will read
+    your comment. Discuss problems internally, if they cannot be addressed
+    in an encouraging way towards the contributor.
+*   If a maintainer sees **additional issues** while reviewing a contribution,
+    these should be addressed in a separate Pull Request, as this is **beyond
+    scope**.
+*   In some areas, **maintainers can be "opinionated"** and may not agree, because
+    there is no objective criteria to discuss. Such a discrepancy should be
+    discussed internally. We are working on a catalogue of "debatable" topics,
+    like for example when to make PHP code "final" or what to make "private/protected".
+    All of our manuals should ideally follow the same lead. Whenever we hit
+    a debatable topic, we should try to find a common ground and document how we
+    decided to deal with this. These decisions are found in :ref:`review-policy-opinionated`.
+
+After a Pull Request has been merged:
+
+*   Check if there were any **labels that require follow-up changes** to the
+    Pull Request (like rework of wording, changing code snippets, adding
+    screenshots). If applicable, a maintainer can either create the Pull
+    Request themself, or maybe ask contributors, if they want to follow up.
+*   If a **backport** is needed, see :ref:`backport-changes` on how to perform this.
+*   The maintainer who merges a Pull Request can make a decision on the scope
+    of the backport. If a backport is too hard or time-consuming to execute,
+    a maintainer may decide to not perform it. Any maintainer who feels otherwise
+    is free to perform the backport on their own.
+*   If a maintainer finds **issues or disagrees with a performed merge** after the fact,
+    this must be discussed internally. A revert is always possible, as is a follow-up
+    PR addressing mistakes. We are humans, and do make those from time to time.
+
+..  _review-policy-opinionated:
+
+"Opinionated" decisions
+=======================
+
+Some decisions can not be made objectively, there may be reasons against and for it.
+Like picking "spaces or tabs" for intendation, or how to apply Uppercase Wording.
+
+Here is a list of currently agreed stances:
+
+*   We use "Sentence case" for all headlines, so only uppercase words at the
+    beginning of a sentence, and following english language uppercasing rules,
+    plus uppercase all special words found in our :ref:`Glossary <spelling-ref>`.
+*   Headline levels are used semantically/structurally, not visually. Their levels
+    must follow logical rules and leave no levels out (for example, no headline of
+    level 4 after a level 1 headline).
+*   In PHP code examples, prefer Dependency Injection over `makeInstance` calls,
+    whenever possible.
+*   PHP Coding Guidelines adhere to the PSR-2 standard, as described in the
```

Comment:
IMO PSR-12/PER-CS1.0
https://docs.typo3.org/m/typo3/guide-contributionworkflow/main/en-us/Appendix/Cgl.html#php
If we need PSR-2 for documentation PHP examples because of specific structures in our automatisms I would remove the link to the TYPO3 Coding Guide Lines. What do you think?

Reply:
Leaving this to @linawolf

Reply:
I think we should mention that we generally follow them but there can be exceptions for didactic or display reasons

Reply:
added.